saleor/graphql/payment/dataloaders.py

```diff
@@ -19,6 +19,16 @@ def batch_load(self, keys):
         return [event_map.get(transaction_id, []) for transaction_id in keys]
 
 
+class TransactionItemsByIDLoader(DataLoader):
```

Comment:
```suggestion
class TransactionItemByIDLoader(DataLoader):
```
as the one `TransactionItem` is returned for each `id`